reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -24,12 +44,17 @@
 
 .heatmap table tr.area td {
   border: 0;
-  padding: 0;
   vertical-align: middle;
 }
 
-.heatmap table tr.area:first-child td {
-  padding-bottom: 10px;
+.area {
+  border-top: 1px solid #f2f2f2;
+}
+
+.darkmode .area {
+  color: #f2f2f2;
+  background-color: rgb(64, 64, 64);
+  border-top-color: 1px solid #020202;
```

Comment:
should only have the colour value

Reply:
✅

Reply:
Did you intend to delete the background-color and border-top-color attributes?

Reply:
no longer using this block